remoteit/desktop
# Device name in logs

### Description
- Adding device name to logs and showing licensing change events
- Adding (deleted) message next to service id display

### Test plan



1. Sign in
2. Go to 'more...' and to 'Logs' page
3. See device name and service name together
4. Register your local device and see the online events in the logs page
5. Unregister (delete) your local device and see the service UIDs
